neqo-client/src/main.rs

```diff
@@ -1022,13 +1022,14 @@ fn main() -> Res<()> {
         }
     }
 
-    let mut urls_by_origin: HashMap<Origin, Vec<Url>> = HashMap::new();
+    let mut urls_by_origin: HashMap<Origin, VecDeque<Url>> = HashMap::new();
     for url in &args.urls {
         let entry = urls_by_origin.entry(url.origin()).or_default();
-        entry.push(url.clone());
+        entry.push_back(url.clone());
     }
```

Comment:
Looking at this, I find this loop annoy, because it could produce what the next loop needs `(host, port) -> Url` more directly.

Reply:
See a0975fcc.